src/jit-pintool/README.md

```diff
@@ -0,0 +1,14 @@
+# clrjit pintool
+
+This directory contains the source code of a pintool that can be used with
+[PIN](https://www.intel.com/content/www/us/en/developer/articles/tool/pin-a-dynamic-binary-instrumentation-tool.html)
+to measure throughput of the JIT. The pintool counts the number of instructions
+executed inside the JIT only. Furthermore it has some special support to
+integrate with SuperPMI's metric collection to allow support for diffing
+throughput.
+
+## Building
+The easiest way to build it is to follow PIN's manual and adding the pintool
+here as another example. See the "Building the Example Tools" and "Building Your
+Own Tool" sections in the manual.
```

Comment:
Can you add a link to the manual?

Reply:
The link is above.

Reply:
For easier discovery, I would do something like:

```suggestion
 ["Building the Example Tools"](https://software.intel.com/sites/landingpage/pintool/docs/98547/Pin/html/index.html#BuildingExamples) and ["Building Your Own Tool"](https://software.intel.com/sites/landingpage/pintool/docs/98547/Pin/html/index.html#BUILDINGTOOLS) sections in the manual.
```

Reply:
I would rather not link to a specific version of a manual. Even manuals from releases of under one year ago are already dead.